examples/time/virtual_time.rs

```diff
@@ -0,0 +1,219 @@
+                    // update the texts on a timer to make them more readable
+                    // `on_timer` run condition uses `Virtual` time meaning it's scaled
+                    // and would result in the UI updating at different intervals based
+                    // on `Time<Virtual>::relative_speed` and `Time<Virtual>::is_paused()`
+                    .run_if(on_real_timer(Duration::from_millis(250))),
+            ),
+        )
+        .run();
+}
+
+/// `Real` time related marker
+#[derive(Component)]
+struct RealTime;
+
+/// `Virtual` time related marker
+#[derive(Component)]
+struct VirtualTime;
+
+/// Setup the example
+fn setup(mut commands: Commands, asset_server: Res<AssetServer>, mut time: ResMut<Time<Virtual>>) {
+    // start with double `Virtual` time resulting in one of the sprites moving at twice the speed
+    // of the other sprite which moves based on `Real` (unscaled) time
+    time.set_relative_speed(2.);
+
+    commands.spawn(Camera2dBundle::default());
+
+    let virtual_color = Color::GOLD;
+    let sprite_scale = Vec2::splat(0.5).extend(1.);
+    let texture_handle = asset_server.load("branding/icon.png");
+
+    // the sprite moving based on real time
+    commands.spawn((
+        SpriteBundle {
+            texture: texture_handle.clone(),
+            transform: Transform::from_scale(sprite_scale),
+            ..default()
+        },
+        RealTime,
+    ));
+
+    // the sprite moving based on virtual time
+    commands.spawn((
+        SpriteBundle {
+            texture: texture_handle,
+            sprite: Sprite {
+                color: virtual_color,
+                ..default()
+            },
+            transform: Transform {
+                scale: sprite_scale,
+                translation: Vec3::new(0., -160., 0.),
+                ..default()
+            },
+            ..default()
+        },
+        VirtualTime,
+    ));
+
+    // info UI
+    let font_size = 40.;
+
+    commands
+        .spawn(NodeBundle {
+            style: Style {
+                display: Display::Flex,
+                justify_content: JustifyContent::SpaceBetween,
+                width: Val::Percent(100.),
+                position_type: PositionType::Absolute,
+                top: Val::Px(0.),
+                padding: UiRect::all(Val::Px(20.0)),
+                ..default()
+            },
+            ..default()
+        })
+        .with_children(|builder| {
+            // real time info
+            builder.spawn((
+                TextBundle::from_section(
+                    "",
+                    TextStyle {
+                        font_size,
+                        ..default()
+                    },
+                ),
+                RealTime,
+            ));
+
+            // keybindings
+            builder.spawn(
+                TextBundle::from_section(
+                    "CONTROLS\nUn/Pause: Space\nSpeed+: +/Up\nSpeed-: -/Down",
```

Comment:
```suggestion
                    "CONTROLS\nUn/Pause: Space\nSpeed+: Up\nSpeed-: Down",
```

(or handle both sets of keys)